Update KIC Prometheus docs (#3288)
Kong/docs.konghq.com: app/kubernetes-ingress-controller/2.0.x/guides/prometheus-grafana.md

## Install Prometheus and Grafana

If you already have the [Prometheus
Operator](https://github.com/prometheus-operator/prometheus-operator) and

> Grafana installed in your cluster, you can skip these steps. Note that you do
> specifically need the Prometheus Operator, as we use its PodMonitor custom
> resource to configure scrape rules.

Reviewer: ```suggestion
Grafana installed in your cluster, you can skip these steps. 

{:.note}
> **Note:** The Prometheus Operator is required, as the {{site.kic_product_name}} 
uses its PodMonitor custom resource to configure scrape rules.
```

### Prometheus and Grafana

First, we will install Prometheus with a
scrape interval of 10 seconds to have fine-grained data points for all metrics.
We’ll install both Prometheus and Grafana in a dedicated `monitoring` namespace.

> First, create a values.yaml to set the scrape interval, use Grafana
> persistence, and install Kong's dashboard:
> 

Reviewer: ```suggestion
Create a `values.yaml` to set the scrape interval, use Grafana
persistence, and install Kong's dashboard:
```

```yaml
prometheus:
  prometheusSpec:
    scrapeInterval: 10s
grafana:
  persistence:
    enabled: true  # enable persistence using Persistent Volumes
  dashboardProviders:
    dashboardproviders.yaml:
      apiVersion: 1
      providers:
      - name: 'default' # Configure a dashboard provider file to
        orgId: 1        # put Kong dashboard into.
        folder: ''
        type: file
        disableDeletion: false
        editable: true
        options:
          path: /var/lib/grafana/dashboards/default
  dashboards:
    default:
      kong-dash:
        gnetId: 7424  # Install the following Grafana dashboard in the
        revision: 5   # instance: https://grafana.com/dashboards/7424
        datasource: Prometheus
  
```

To install Prometheus and Grafana, execute the following, specifying the path

> to the values.yaml you created earlier:

Reviewer: ```suggestion
to the `values.yaml` you created earlier:
```

$ helm repo add kong https://charts.konghq.com
$ helm repo update

$ kubectl create namespace kong
$ helm install mykong kong/kong --namespace kong --set serviceMonitor.enabled=true --set serviceMonitor.labels.release=promstack
```

By default, kube-prometheus-stack [selects ServiceMonitors and PodMonitors by a

> `release` label equal to the release name](https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-19.0.1/charts/kube-prometheus-stack/values.yaml#L2128-L2169). The labels setting here adds a label matching the

Reviewer: ```suggestion
`release` label equal to the release name](https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-19.0.1/charts/kube-prometheus-stack/values.yaml#L2128-L2169). The `labels` setting here adds a label matching the
```

> `promstack` release name from the example earlier.

Reviewer: ```suggestion
`promstack` release name from the example.
```
